app/views/static_pages/techhire_locations.html.erb

```diff
@@ -1 +1 @@
-<div class="map"></div>
+<%=tag("div", class: "map", :data => {:companies => @companies, :coordinates => @companies.map {|company| company.d3_coordinates}})%>
```

Comment:
Data attributes were how I passed the Ruby collection from the controller to the javascript file. I had jquery obtain the data attribute from this div. This was key to rendering the marks on the map.